src/backend/joanie/core/serializers.py

```diff
@@ -426,6 +426,7 @@ def update(self, instance, validated_data):
         The "failed" state can only be set by the LMSHandler.
         """
         validated_data.pop("course_run", None)
+        validated_data.pop("was_created_by_order", None)
```

Comment:
Actually this field should just be readonly all the time.

Reply:
Nope for case where a course is listed and also linked to a product the client has to provide a value for this flag.

Reply:
ok